controllers/admin/AdminImportController.php

```diff
@@ -136,6 +136,7 @@ public function __construct()
                     'ecotax' => array('label' => $this->trans('Ecotax', array(), 'Admin.Catalog.Feature')),
                     'quantity' => array('label' => $this->trans('Quantity', array(), 'Admin.Global')),
                     'minimal_quantity' => array('label' => $this->trans('Minimal quantity', array(), 'Admin.Advparameters.Feature')),
+                    'low_stock_threshold' => array('label' => $this->trans('Low stock level', array(), 'Admin.Advparameters.Feature')),
```

Comment:
Isn't it a string we also have in the Product page? I'd rather use the same domain to avoid duplicates. Please use Admin.Catalog.Feature is this is indeed the case. Thanks!